budgetApp1/budgetApp1/BudgetViewController.swift

```diff
@@ -23,9 +23,7 @@ class BudgetViewController: UIViewController {
         super.viewDidLoad()
         
         viewModel = BudgetViewModel(repository: BudgetRepository(), delegate: self)
-        viewModel?.getBudgets()
-        // Do any additional setup after loading the view.
-         
+        viewModel?.fetchBudgets()
     }
 
     private func fillUI() {
```

Comment:
this name is a bit misleading. Try Avoid UI. Apple uses UI for UIKit classes. so fillUI made me immediately think it was something to do with a UI class.  
Perhaps use setUpView. 
and this is still logic. Create functions in the viewModel please.
and please make the IBOutlets private

Reply:
Done